jenkins/job-configs/kubernetes-jenkins-pull/testinfra-pull.yaml

```diff
@@ -55,6 +56,12 @@
             timeout: 600
             fail: true
     builders:
+        - ensure-upload-to-gcs-script:
```

Comment:
conveniently, it's dumb and won't see the test-infra one - https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins-pull/global.yaml#L15